ui/app/AppLayouts/Onboarding/views/KeysMainView.qml

```diff
@@ -150,10 +150,10 @@ Item {
                 opacity: 0.0
             }
             // TODO: Functionality is not completed. Missing saving of keystore files which blocks wallet account creation. @see #7867
```

Comment:
TODO still valid?

Reply:
@caybro no, it isn't, forgot to remove it, removed now.